### PR TITLE
Drop tags when adding/copying template questions

### LINKED ISSUE #10243: QuestionAddEditor adds `mwest` and `fa17` tags
Per https://github.com/PrairieLearn/PrairieLearn/pull/10221#discussion_r1680150680, since we copy from a reference question, all questions created using a `QuestionAddEditor` adds the tags that the reference question has, including `mwest` and `fa17`. As these tags likely do not apply to new questions, they should not be copied over.

### PR DESCRIPTION
**Pull request by nwalters512**
Closes #10243.

### CODE REVIEW

File: apps/prairielearn/src/lib/editors.ts [hunk 1]
```diff
@@ -1132,6 +1134,12 @@ export class QuestionTransferEditor extends Editor {
     infoJson.title = questionTitle;
     infoJson.uuid = this.uuid;
 
+    // When transferring a question from an example/template course, drop the tags. They
+    // are likely undesirable in the template course.
+    if (this.course.example_course || this.course.template_course) {
+      delete infoJson.tags;
+    }
```
Comment (nwalters512) [inline apps/prairielearn/src/lib/editors.ts:1141]: This is for `QuestionTransferEditor`. We could always unconditionally drop tags? I'm not sure what's best here.

File: apps/prairielearn/src/lib/editors.ts [hunk 2]
```diff
@@ -888,6 +888,8 @@ export class QuestionAddEditor extends Editor {
     debug('Write info.json with new title and uuid');
     infoJson.title = names.longName;
     infoJson.uuid = this.uuid;
+    // The template question contains tags that shouldn't be copied to the new question.
+    delete infoJson.tags;
```
Comment (jonatanschroeder) [inline apps/prairielearn/src/lib/editors.ts:892]: I think the placeholder `"tags"` should be kept...
```suggestion
    infoJson.tags = [];
```

File: apps/prairielearn/src/lib/editors.ts (hunk 1 above)
Comment (mwest1066) [reply apps/prairielearn/src/lib/editors.ts:1141]: I'm inclined to leave them for now. That is, I like your current code.

**Review by mwest1066 — APPROVED** [review]

File: apps/prairielearn/src/lib/editors.ts (hunk 2 above)
Comment (nwalters512) [reply apps/prairielearn/src/lib/editors.ts:892]: IMO it should not be, just as we don't include other "placeholder" values in question `info.json` files.

**Pull request merged**